pkg/server/health_check.go

```diff
@@ -14,7 +17,7 @@ func (s Server) CheckCEDAREasiClientConnection(client cedareasi.TranslatedClient
 	s.logger.Info("Testing CEDAR EASi Connection")
 	// FetchSystems is agnostic to user, doesn't modify state,
 	// and tests that we're authorized to retrieve information
-	_, err := client.FetchSystems(s.logger)
+	_, err := client.FetchSystems(appcontext.WithLogger(context.Background(), s.logger))
```

Comment:
This is the ugliest change I made in this PR... It's because there's no context sent in to this during setup phase of the app. It works, but it'll have to be later cleanup that makes this nicer.